tests/test_api_requests.py

```diff
@@ -209,7 +210,7 @@ async def test_coro_handling(meta: Dict[str, Dict[str, Any]], mockserver):
         ),
         (
             {"zyte_api": {"browserHtml": True, "httpResponseBody": True}},
-            IgnoreRequest,
+            RequestError,
```

Comment:
Do you know why the test above is still checking for IgnoreRequest?

Reply:
The other test handles a catch-all scenario for errors not handled by `python-zyte-api`: https://github.com/scrapy-plugins/scrapy-zyte-api/blob/main/scrapy_zyte_api/handler.py#L151-L155

Reply:
The other test handles a catch-all scenario for errors not handled by `python-zyte-api`: https://github.com/scrapy-plugins/scrapy-zyte-api/blob/main/scrapy_zyte_api/handler.py#L151-L155

Reply:
Why are these errors raised as IgnoreRequest?

Reply:
I believe it's for the same reason to make the logs nicer and prevent the long tracebacks from occurring.

Should we change this to something else as well? I was thinking of having another catch statement that raises `aiohttp.ClientError`.

Reply:
It seems IgnoreRequest is not a right solution for silencing the exceptions in logs anyways, and should be removed. We can make it nicer, but re-raising the original exception looks ok.

Reply:
Removed all usage of `scrapy.exceptions.IgnoreRequest` in https://github.com/scrapy-plugins/scrapy-zyte-api/pull/42/commits/32018f2993343014ee38398ef39260c9945eb570